app/views/notifications/_comment.html.erb

```diff
@@ -1,5 +1,6 @@
 <% commentable = comment.commentable %>
-<% comment_path = polymorphic_path([@project, commentable], anchor: dom_id(comment)) %>
+<% project = commentable.node.project %>
```

Comment:
I don't get this `commentable.node.project`.
What if in the future we want to comment on a node?
I think this fixes the problem now, but this is not generic.
How to fix this?

Reply:
If we don't want to add `project_id` to `comments` or to `issues`, may be we could add a `project` method to the `Issue`model

Reply:
I'm going with adding a `project` method (delegation) to the `Note` and `Issue` model same as in Pro